shub/image/build.py

```diff
@@ -39,18 +39,21 @@
               help="stream build logs to console")
 @click.option("-V", "--version", help="release version")
 @click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
-def cli(target, debug, verbose, version, skip_tests):
-    build_cmd(target, version, skip_tests)
+@click.option("-f", "--file", "filename",
+              help="Name of the Dockerfile (Default is 'PATH/Dockerfile')")
+def cli(target, debug, verbose, version, skip_tests, filename):
+    build_cmd(target, version, skip_tests, filename=filename)
 
 
-def build_cmd(target, version, skip_tests):
+def build_cmd(target, version, skip_tests, filename=None):
     config = load_shub_config()
     create_scrapinghub_yml_wizard(config, target=target, image=True)
     client = utils.get_docker_client()
     project_dir = utils.get_project_dir()
     image = config.get_image(target)
     image_name = utils.format_image_name(image, version)
-    if not os.path.exists(os.path.join(project_dir, 'Dockerfile')):
+    dockerfile = filename or 'Dockerfile'
```

Comment:
(Together with the default value for the click option we could set `filename='Dockerfile'` in the parameter list above and avoid this line.)